app/helpers/form_helper.rb

```diff
@@ -66,4 +66,9 @@ def us_states_territories
       ['Wyoming', 'WY'],
     ]
   end
+
+  # Filtered list to removve territories that error due to USPS API
+  def filtered_us_states_territories
+    us_states_territories.filter { |t| !['AA','AE','AP','UM'].include? t[1] }
```

Comment:
1. I have a personal beef with un-named array access, I would name the values in the block params
2. I would consider `%w[]` syntax and the optional parens for readability
3. The leading`!` is easy to skim over, maybe `reject` would be clearer here

```suggestion
    us_states_territories.reject { |_name, abbrev| %w[AA AE AP UM].include?(abbrev) }
```

Reply:
Oh these changes are awesome. Especially if we are moving it to a different file. Will commit these, thanks!